src/mixins/test/wordWrap.test.js

```diff
@@ -0,0 +1,14 @@
+import wordWrap from '../wordWrap'
+
+describe('wordWrap', function() {
+
+  it('should accept other values', function() {
+    expect(wordWrap('break-all')).toMatchSnapshot()
+  })
+
+  it('should default wrap to break-word', function() {
+    expect(wordWrap()).toMatchSnapshot()
+  })
```

Comment:
@mxstbr Let's change it now, so it is one less thing we will need to go back and change.